tasks/src/reinforcement-learning/about.md

```diff
@@ -52,7 +57,77 @@ Observations and states are the information our agent gets from the environment.
 
 ## Inference
 
-You can add a small snippet [here](https://github.com/huggingface/hub-docs/blob/main/tasks/src/reinforcement-learning/about.md) that shows how to infer with `reinforcement-learning` models.
+Inference in Reinforcement learning, differs a bit from traditional supervised learning where you just deal with model and test data. In RL, once you have trained an agent in an env, you try to run the trained agent for specific no of steps to get the average reward its able to get. 

```

Comment:
```suggestion
Inference in reinforcement learning differs from other modalities, in which there's a model and test data. In reinforcement learning, once you have trained an agent in an environment, you try to run the trained agent for additional steps to get the average reward. 

```
I feel like you're mentioning about evaluation of agents here and not directly inference, am I correct?

Reply:
yes, i'll add a code snippet for inference as well. missed out on that